C/primitive/elements/jets.c

```diff
@@ -274,6 +300,28 @@ bool input_issuance_token_amt(frameItem* dst, frameItem src, const txEnv* env) {
   return true;
 }
 
+/* input_issuance_asset_proof : TWO^32 |- S (S TWO^256) */
```

Comment:
In d2063b4938c2ecdddd1c855ae930e46cd2a5ce22:

This comment should have `ONE` as the source rather than `TWO^32` (same with the immediately following function).

Reply:
Actually this same mistake occurs in the auto-generated bindings and in the Haskell -- I can't remember now what data the generated code is produced from, but I guess the typo is in there.

Reply:
The input is the index of the input to access.  As you can see below

    uint_fast32_t i = read32(&src);

Reply:
Fixed in the other location.